tests/test_citations.py

```diff
@@ -0,0 +1,174 @@
+import datetime
+from nose.tools import *
+
+from scripts import parse_citation_styles
+from website import citations
+from website.util import api_url_for
+from website.citations.utils import datetime_to_csl
+from website.models import Node, User
+
+from tests.base import OsfTestCase
+from tests.factories import ProjectFactory, UserFactory
+from tests.test_features import requires_csl_styles
+
+
+bibtex_template = '''@misc{{{id},
+  title={{{title}}},
+  url={{{url}}},
+  publisher={{Open Science Framework}},
+  author={{{authors}}},
+  year={{{year}}},
+  month={{{month}}}
+}}'''
+
+
+class CitationsTestCase(OsfTestCase):
+    def setUp(self):
+        self.node = ProjectFactory()
```

Comment:
Always call the the superclass's `setUp` and `tearDown` when overriding these methods.

http://cosdev.readthedocs.org/en/latest/osf/testing.html